crates/cargo-test-support/src/compare.rs

```diff
@@ -201,6 +201,18 @@ static MIN_LITERAL_REDACTIONS: &[(&str, &str)] = &[
     ("[EXE]", std::env::consts::EXE_SUFFIX),
     ("[BROKEN_PIPE]", "Broken pipe (os error 32)"),
     ("[BROKEN_PIPE]", "The pipe is being closed. (os error 232)"),
+    // Unix message for an entity was not found
+    ("[NOT_FOUND]", "No such file or directory (os error 2)"),
+    // Windows message for an entity was not found
+    (
+        "[NOT_FOUND]",
+        "The system cannot find the file specified. (os error 2)",
+    ),
+    (
+        "[NOT_FOUND]",
+        "The system cannot find the path specified. (os error 3)",
+    ),
+    ("[NOT_FOUND]", "program not found"),
```

Comment:
Do you know under what situation os error 2 vs 3 shows uo?

Reply:
I must admit that I don't have any idea about this. I completely agree with what @weihanglo  says in https://github.com/rust-lang/cargo/pull/14068/files#r1649254235 - it feels like a whack-a-mole game on Windows 🙈